kubernetes/loki.yaml

```diff
@@ -4,10 +4,9 @@ apiVersion: policy/v1beta1
 kind: PodSecurityPolicy
 metadata:
   name: loki
-  namespace: app-loki
```

Comment:
is this important?

Reply:
Honestly not sure, this seems just like the result of bumping the chart